spec/requests/drafts_controller_spec.rb

```diff
@@ -24,4 +24,15 @@
     parsed = JSON.parse(response.body)
     expect(parsed["drafts"].length).to eq(0)
   end
+
+  it 'does not let userA see drafts by userB' do
+    userB = Fabricate(:user)
+    Draft.set(userB, 'xxx', 0, '{}')
+
+    userA = sign_in(Fabricate(:user))
```

Comment:
We don't need this variable here :) On the ruby side, we also prefer to use snake_case instead of camelCase.